.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Description:
+_Summary of change, including the issue(s) to be adressed._
+
+## Type of change:
+_Put an `x` in the box that applies._
+- [ ] Bug fix.
+- [ ] New feature.
+- [ ] Documentation update.
+
+## Checklist:
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._
+
+This checklist can be used as a help for the reviewer.
```

Comment:
```suggestion
<!-- This checklist can be used as a help for the reviewer. -->
```
Maybe you want this shown in the PR though?

Reply:
Yes, this should stay visible I think.